### PR TITLE
.java filenames matched with public class

### PR DESCRIPTION
**Pull request by jarroyo3**
It have been created a function to check if language is Java to change the filename to  public class name. #47

### CODE REVIEW

File: hackIDE/static/hackIDE/js/custom.js [hunk 1]
```diff
@@ -269,7 +290,13 @@ $(document).ready(function(){
 						$(".output-io-info").hide();
 						$(".compile-status").children(".value").html("--");
 						$(".error-key").html("Compile error");
-						$(".error-message").html(response.compile_status);
+
```
Comment (jarroyo3) [inline hackIDE/static/hackIDE/js/custom.js]: I'm sorry. I forgot delete this piece of code.

I wrote this because when I compile a Java program (for example) and the code is like some "sdasdas", the compile error throws an empty error.

Comment (sahildua2305) [inline hackIDE/static/hackIDE/js/custom.js]: I didn't understand this. What did you intend to do here?

Comment (jarroyo3) [inline hackIDE/static/hackIDE/js/custom.js]: I will try to explain it.

on language selector, I choose Java. If I write for example

public class TestDriver { 

a close bracket (}) is missing. if you try to compile the code the error mesaage thown is empty.

The cause of that is because ajax response variable (compile_status) contains a blank space.

Those changes are mandé un línea 294

Comment (sahildua2305) [inline hackIDE/static/hackIDE/js/custom.js]: @jarroyo3 would you be willing to separate these two changes in two different PRs? That would be helpful in easy reviewing and managing the code commits history :)

Comment (jarroyo3) [inline hackIDE/static/hackIDE/js/custom.js]: Hi!

Sure! How should I proceed? Two different branches with two different commits?

Should I create other branch solving the "empty error"?

Comment (sahildua2305) [inline hackIDE/static/hackIDE/js/custom.js]: Hi @jarroyo3, yes. You can create another branch for "empty error" problem.

**Comment by sahildua2305** [comment]
Thanks @jarroyo3 for your contributions. Looks good to me 💯
